ingest/src/main/java/com/connexta/ingest/client/StoreClient.java

```diff
@@ -69,12 +68,7 @@ public String getFilename() {
         new HttpEntity<>(builder.build(), headers);
     log.info("Sending POST request to {}: {}", storeEndpoint, request);
 
-    final URI location;
-    try {
-      location = restTemplate.postForLocation(storeEndpoint, request);
-    } catch (final RestClientException e) {
-      throw new StoreException("Unable to POST to store endpoint", e);
-    }
-    return location;
+    return new StreamingRestTemplate(storeEndpoint, HttpMethod.POST)
```

Comment:
❓ Can we share the same RestTemplate instead of instantiate a new one for every store request?